
#fail= #intentional fail

def myfunc():
    return 1


